modules/capability/go.mod

```diff
@@ -5,14 +5,14 @@ go 1.21
 replace github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 
 require (
-	cosmossdk.io/core v0.10.0
+	cosmossdk.io/core v0.11.0
```

Comment:
Would be nice to isolate this in a separate PR so that we can backport it cleanly

Reply:
Done https://github.com/cosmos/ibc-go/pull/4689